src/common/util/src/monitors/cpu_performance_counter.cpp

```diff
@@ -0,0 +1,206 @@
+// Copyright (C) 2019-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/util/monitors/cpu_performance_counter.hpp"
+
+#include <algorithm>
+#include <iostream>
+#include <map>
+
+#ifdef _WIN32
```

Comment:
Maybe we can split this file to multiple files(merge with query_wrapper), each one for one platform(Window, Linux).

Reply:
will update based on this suggestion.